lib/Sanger/CGP/CavemanPostProcessor.pm

```diff
@@ -624,15 +656,21 @@ sub _callbackMatchedNormFetch{
       $str = -1;
     }
     my $cig_str = $algn->cigar_str; # expensive and reused so save to variable
-
     #Read population
     $this_read->{str} = $str;
-    $this_read->{qbase} = substr $a->qseq, $rdPosIndexOfInterest-1, 1;
+
+    #if($is_covered == 1){
```

Comment:
Commented `if` block, can this be removed?  Makes it look like it was in the middle of investigation.